pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -62,6 +64,32 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 		if err != nil {
 			return nil, err
 		}
+
```

Comment:
If we moved this logic outside of the create block, that would let us change the value after-the-fact, right?